bin/init.sh

```diff
@@ -112,6 +114,15 @@ if [ ! -f ${ISTIO_OUT}/envoy ] ; then
     cp $OUT/envoy-$PROXYVERSION ${ISTIO_OUT}/envoy
 fi
 
+if [ ! -f /usr/local/bin/helm -a ! -f ${ISTIO_OUT}/helm ] ; then
+    # Install helm. Please keep it in sync with .circleci
+    cd /tmp && \
+        curl -Lo /tmp/helm.tgz https://storage.googleapis.com/kubernetes-helm/helm-${HELM_VER}-linux-amd64.tar.gz && \
```

Comment:
try vendor ?
let's not add more stuff that only works on linux for no good reason